index.html

```diff
@@ -200,5 +200,38 @@ <h2>About me</h2>
         </div>
     </section>
 
+    <form action="https://formspree.io/f/xrgvbpnv" method="POST">
+        <ul>
+            <li>
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </li>
+            <li>
+                <input type="text" name="Full name" maxlength="30" placeholder="Full name" required/>
+            </li>
+            <li>
+                <input type="email" name="Email address" placeholder="Email address" required/>
+            </li>
+            <li>
+                <textarea name="Text Area" id="txtArea" maxlength="500" placeholder="Enter Text Here"required></textarea>
+            </li>
+        </ul>
+        
+        <center><button type="submit">Get in touch</button></center>
+        
+    </form>
+    <div class="footer-img"><img src="./Images//illustration-contact-form.svg" alt=""></div>
+    <footer>
+        <ul class="footer-socials">
+            <li><a href="#"><img src="./Images//github.svg" alt=""></a></li>
+            <li><a href="#"><img src="./Images//linkedin.svg" alt=""></a></li>
+            <li><a href="#"><img src="./Images//angellist.svg" alt=""></a></li>
+            <li><a href="#"><img src="./Images//twitter.svg" alt=""></a></li>
+            <li><a href="#"><img src="./Images//medium.svg" alt=""></a></li>
```

Comment:
- [Optional] Kindly make sure you used the exact file name and path for your image because it is not visible in some environments, 👌😁 your image folder is `image` not `Image` and don't use `//` when you write path use `/` 😢, I would prefer to use small case for all static file and directories 👌👍